### ISSUE TITLE
"Conduct test" page's empty queue message is marked as a list

### ISSUE THREAD

**Issue by emyl3** [CDCgov/prime-simplereport#8549]
## Area or Component

`/app/queue` "Conduct test" page - when you have an empty queue

## Quick Summary

Content is not a list but it is marked as such.

### How to Replicate
1. Go to `/app/queue`
2. Have an empty queue
3. Notice that the following message: "There are no tests running. Search for a patient to start their test. To add results in bulk for COVID-19, Flu A and B, and RSV using a CSV file, go to Upload spreadsheet." is marked up as an unordered list when the content is not a list. 

## Accessibility Violation

[semantic-list-used-for-presentation
](https://docs.deque.com/issue-help/1.0.0/en/semantic-list-used-for-presentation)

## Action Requested

When there is an empty test queue, ensure the message is not an `<ul>`.

## Screenshots (optional)

<img width="2000" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBd2NSQ1E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--4a658ea2e58b909cd07c565be66b8b50b0ad5504/Screenshot%202025-02-28%20at%2007.15.44.png" alt="Screenshot 2025-02-28 at 07.15.44.png" />

## Source Code (optional)

[which file or line is the issue attached to?]: #
